src/V3Width.cpp

```diff
@@ -4964,7 +4969,7 @@ class WidthVisitor final : public VNVisitor {
             nodep->v3error("Assignment pattern with no members");
         }
         // UINFOTREE(9, newp, "", "apat-out");
-        VL_DO_DANGLING(pushDeletep(nodep), nodep);  // Deletes defaultp also, if present
+        VL_DO_DANGLING(pushDeletep(nodep), nodep);
```

Comment:
The comments here are all lies. `defaultp` was unlinked fromp the `nodep` so it survives. If you are looking for a really tedious job, fixing the leaks in here might be a good one :sweat_smile: